### PR TITLE
Small word list changes

### PR DESCRIPTION
**Pull request by bethanyaconnor**
Two columns of words: colors and shapes. The options in each column are randomized.
Screenshot 2019-11-12 at 9 58 58 AM

### CODE REVIEW

File: src/demo/ui.jsx [hunk 1]
```diff
@@ -543,8 +546,26 @@ class Words extends React.Component {
   constructor(props) {
     super(props);
 
-    // Randomize word choices and set in state.
-    const choices = _.shuffle(wordSet[getState().appMode].choices);
+    // Randomize word choices in each set, merge the sets, and set as state.
+    const appMode = getState().appMode;
+    const appModeWordSet = wordSet[appMode].choices;
+    let choices = [];
+    let maxSize = 0;
```
Comment (uponthesun) [inline src/demo/ui.jsx]: Do we need to generalize this to the case where there are a different number of entries in each column? If we don't foresee needing it we could just instead validate that they're all equal to simplify.

File: src/demo/ui.jsx [hunk 2]
```diff
@@ -543,8 +546,26 @@ class Words extends React.Component {
...
+    // Each subset represents a different column, so merge the sets
+    for (var i = 0; i < appModeWordSet.length; ++i) {
+      appModeWordSet[i] = _.shuffle(appModeWordSet[i]);
+      if (appModeWordSet[i].length > maxSize) {
+        maxSize = appModeWordSet[i].length;
+      }
+    }
+    for (i = 0; i < maxSize; ++i) {
```
Comment (uponthesun) [inline src/demo/ui.jsx]: What's the reason for this specific nested loop - does `choices` need to be in a certain order?

Comment (bethanyaconnor) [reply src/demo/ui.jsx]: Yeah. The colors all need to be in the left column and the shapes in the right.

Comment (uponthesun) [reply src/demo/ui.jsx]: But the local variable `choices` that you set in state is just a flat list which is mixed between the two, right?

Comment (bethanyaconnor) [reply src/demo/ui.jsx]: In order to get it to show up correctly it needs to be separated into the two columns. I was trying to figure out the CSS to do this "better" but couldn't maintain the button styles correctly.

File: src/demo/ui.jsx (hunk 1 above)
Comment (bethanyaconnor) [reply src/demo/ui.jsx]: Fair enough. I tend to err on the side of handling array out of bounds (and like errors) even if we don't think they'll be necessary. I'll add a comment here.

File: src/demo/ui.jsx (hunk 2 above)
Comment (madelynkasula) [reply src/demo/ui.jsx]: the order only matters for display reasons, so i think it's okay that the end result is a flat list

Comment (uponthesun) [reply src/demo/ui.jsx]: 👍 bethany explained it to me on slack.

**Pull request merged**